generators/app/templates/src/app/i18n.ejs

```diff
@@ -12,6 +12,10 @@ i18next.addResources('es', 'app', {
     [Routes.Tab1]: 'Tab 1',
     [Routes.Tab2]: 'Tab 2',
     <%_ } _%>
-    [Routes.Home]: 'Home'
+    [Routes.Home]: 'Home'<%_ if(features.onboarding) { _%>,
+    [Routes.OnBoarding]: 'OnBoarding'
```

Comment:
is this i18n file working good? I thing the `es: {` shouldn't be there

Reply:
It was always like that. I have to check

Reply:
And yes! You were right!